modules/oe_webtools_media/src/Plugin/media/Source/Webtools.php

```diff
@@ -124,22 +124,30 @@ public function getWidgetTypes() {
     return [
       'chart' => [
         'name' => $this->t('Chart'),
-        'service' => 'charts',
+        // @deprecated Use services key instead.
```

Comment:
This is not good. My comment said: ` So we need to leave service in place with charts`. You removed `charts` hence broke BC...